src/SprykerEco/Yves/Braintree/Form/DataProvider/CheckoutShipmentFormDataProvider.php

```diff
@@ -218,7 +218,7 @@ protected function getDeliveryTime(ShipmentMethodTransfer $method): int
     protected function getFormattedShipmentPrice(ShipmentMethodTransfer $shipmentMethodTransfer): string
     {
         $moneyTransfer = $this->moneyPlugin
-            ->fromInteger($shipmentMethodTransfer->getStoreCurrencyPrice());
+            ->fromInteger((int)$shipmentMethodTransfer->getStoreCurrencyPrice());
```

Comment:
same question

Reply:
The reason for typecasting here is that phpstan complains about possible `null` returning from `getStoreCurrencyPrice()`. Not sure how to treat `null` values here. Should it be outputted as 0 price (which is what the typecast results in now), or price should not be shown at all in case of `null`?

Reply:
Fixed